docs/sql/mysql/create_push_server_schema.sql

```diff
@@ -68,6 +67,12 @@ CREATE TABLE `push_inbox` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1 CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
+-- Create table for assignment of inbox messages to apps
+CREATE TABLE `push_inbox_app` (
```

Comment:
The table has no primary key. Usually database tables do have a primary key, unless it is just a dump of data.

Reply:
Agreed.